deploy/1-deploy-oracles.ts

```diff
@@ -8,14 +8,16 @@ const ADDRESSES = {
   bsctestnet: {
     vBNBAddress: testnetDeployments.Contracts.vBNB,
     WBNBAddress: "0xae13d989daC2f0dEbFf460aC112a837C89BAa7cd",
+    VAIAddress: "0x05889b75Ae9Aa53AaE49562432201a0c93BaCA0b",
```

Comment:
I think `0x05889b75Ae9Aa53AaE49562432201a0c93BaCA0b` is a very old VAI token in testnet. We can use the included in the core package ([0x5fFbE5302BadED40941A403228E6AD03f93752d9](https://testnet.bscscan.com/address/0x5fFbE5302BadED40941A403228E6AD03f93752d9)), that seems newer.

```suggestion
    VAIAddress: testnetDeployments.Contracts.VAI,
```

Reply:
Fixed